modules/yaml-test.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+:_content-type: CONCEPT
+[id="REPLACE_ME_WITH_ID_{context}"]
+= Included YAML example
+
+Have a look at this YAML.
+
+[source,yaml]
+----
+include::yaml/install-config.yaml[]
+----
+<1> Instance type, for example, `m5d.2xlarge`.
```

Comment:
Rather than callouts here, I would maybe put these details as comments directly in the YAML file itself.

Reply:
I guess it would make it much more useable from a code review perspective.

Reply:
Yeah, much more usable for reviewers. Plus the fact callouts are an asciidoc thing so don't belong in yaml or other types.

The one consideration here is that embedding comments in the YAML means that users copy the comments along with the YAML. That's kind of a double-edged sword. I think as long as the comments are meaningful and concise then it shouldn't be an issue.

Reply:
Yeah and this is where I'm torn. For example: https://docs.openshift.com/container-platform/4.11/networking/using-ptp.html#configuring-linuxptp-services-as-boundary-clock_using-ptp.

I absolutely take your point though. Comments in the YAML work for the end user too I think.

Reply:
Yeah that example is tough. That's too much YAML for a procedure and all those callouts essentially nest a massive chunk of reference content.

Another point to mention here is around personas, their needs, and their knowledge. From my perspective, the following bit is self-explanatory and the existing callout (`#3` in this PR) is silly noise that actually detracts from the docs being authoritative.

```
sshKey: |
  <ssh_key>
 ```
 
 If you have a descriptive example value, you really don't need a callout. IMO callouts are best used to provide canonical examples within concepts.

Reply:
:) I take your point - these examples are not real world. I just wanted to test out and see if commented callouts (`# <3>`) would be rendered in the build.